### ISSUE TITLE
Y axis has a wrong range in a chart with big amount of data

### ISSUE THREAD

**Issue by limion** [highslide-software/highcharts.com#4513]
Here is an example: http://jsfiddle.net/qgoe88kh/
Data table is here: http://proagro.com.ua/chartsmanager/chart/view?id=135

As we can see the maximum Y value is 4800, but Y axis has a range -5K to 20K

**Comment by pawelfus** [comment]
I think this is correct, let me explain:

- each of the highest points is value around 5k
- tooltip displays value of the point
- you have stacking, so each line is rendered as sum of previous values
- tooltip still displays point value, not the sum
- you have four series, so when you stack them, then max on yAxis is around 18k
- 18k is rounded to 20k to get rounded ticks, like 0k, 5k, 10k etc.

In other words, maybe you just want to disable stacking: http://jsfiddle.net/qgoe88kh/1/ ?

**Comment by limion** [comment]
Yes, you are right, i did not noticed stacking option. Sorry
